tests/acceptance/features/apiGraphUserGroup/getUser.feature

```diff
@@ -1437,160 +1438,236 @@ Feature: get users
+                },
+                {
+                  "type": "object",
+                  "required": [
+                    "displayName",
+                    "id",
+                    "userType"
+                  ],
+                  "properties": {
+                    "displayName": {
+                      "type": "string",
+                      "const": "Alice Hansen"
+                    },
+                    "id": {
+                      "type": "string",
+                      "pattern": "^%user_id_pattern%$"
+                    },
+                    "userType": {
+                      "type": "string",
+                      "const": [
+                        "Admin"
+                      ]
+                    }
                   }
                 }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+  @env-config
+  Scenario Outline: search other users when OCIS_SHOW_USER_EMAIL_IN_RESULTS config is enabled
+    Given the config "OCIS_SHOW_USER_EMAIL_IN_RESULTS" has been set to "true"
+    And the administrator has assigned the role "<user-role>" to user "Alice" using the Graph API
+    When user "Alice" searches for user "Brian" using Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "value"
+        ],
+        "properties": {
+          "value": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 1,
+            "items": {
+              "type": "object",
+              "required": [
+                "displayName",
+                "id",
+                "userType",
+                "mail"
+              ],
+              "properties": {
+                "displayName": {
+                  "type": "string",
+                  "const": "Brian Murphy"
+                },
+                "id": {
+                  "type": "string",
+                  "pattern": "^%user_id_pattern%$"
+                },
+                "userType": {
+                  "type": "string",
+                  "const": "Member"
+                },
+                "mail": {
+                  "type": "string",
+                  "const": "brian@example.org"
+                }
               }
-            ]
+            }
           }
         }
       }
-    }
-    """
+      """
+    Examples:
+      | user-role   |
+      | Admin       |
+      | Space Admin |
+      | User        |
+      | User Light  |
+
+
+  @env-config
+  Scenario Outline: search other users when OCIS_SHOW_USER_EMAIL_IN_RESULTS config is disabled
+    Given the config "OCIS_SHOW_USER_EMAIL_IN_RESULTS" has been set to "false"
+    And the administrator has assigned the role "<user-role>" to user "Alice" using the Graph API
+    When user "Alice" searches for user "Brian" using Graph API
+    Then the HTTP status code should be "200"
```

Comment:
this also should be covered in above scenarios

Reply:
moved scenario to `searchUserIncludingEmail.feature`